datastore-design/go-grpc-demo/ts-init.sql

```diff
@@ -14,7 +14,7 @@ CREATE TABLE time_series (
 
 CREATE TABLE observations (
 	ts_id integer REFERENCES time_series(id) ON DELETE CASCADE,
-	tstamp timestamp, -- obs time (NOT NULL, but implied by being part of PK)
+	tstamp timestamptz, -- obs time (NOT NULL, but implied by being part of PK)
```

Comment:
Can we assume UTC globally and thus avoid time zones (i.e. still use the `timestamp` type)? (at MET all observation and forecast times are represented internally as UTC, i.e. leaving it to external consumers/clients to convert to local times as required)

Reply:
We should indeed always put in UTC times. But when I looked at this earlier, it is recommended to always use timestamptz in Postgres. There is no storage overhead for this.

I don't remember exactly what the reasoning was (I can look it up), but I think the idea is that at least the server then knows that the timestamps are in UTC.

Reply:
Makes sense.